Makefile

```diff
@@ -40,19 +40,27 @@ PLATFORM ?= linux/$(GOARCH)
 BUILD_ARGS ?=
 
 # Image tag for controller image build
-CONTROLLER_TAG ?= cosi-controller:latest
+CONTROLLER_TAG ?= localhost:5005/cosi-controller:dev-$(shell git describe --match='' --always --abbrev=6 --dirty)
 
 # Image tag for sidecar image build
-SIDECAR_TAG ?= cosi-provisioner-sidecar:latest
+SIDECAR_TAG ?= localhost:5005/cosi-provisioner-sidecar:dev-$(shell git describe --match='' --always --abbrev=6 --dirty)
 
 ##@ Development
 
 .PHONY: generate
 generate: controller/Dockerfile sidecar/Dockerfile ## Generate files
 	$(MAKE) -C client crds
 	$(MAKE) -C proto generate
+	$(MAKE) api-docs
```

Comment:
I'm curious why this is `api-docs` instead of `build-docs`. I guess I am confused how they are conceptually different. Is `build-docs` only needed in order to `serve-docs`, or is it used to generate files that should be checked in?

Reply:
Agreed, I’ll merge the two targets into one! Thanks.

Reply:
Fixed.